autonomy/cli/helpers/analyse.py

```diff
@@ -338,6 +351,10 @@ def _get_chained_abci_skill(
         if skill_config.is_abstract:
             continue
 
+        # Check if the skill has the `abstract_round_abci` skill as a dependency
+        if not _has_abstract_round_abci_skill_as_dependency(skill_config=skill_config):
+            continue
```

Comment:
Makes the abci skill filter even stricter so we don't confuse every skill with `is_abstract` set to `false` as a chained abci skill